tibo-mllr/activeviam-projet
# contributing on issue #23 : changing URL format

# Issue Number:

Fix issue #23 

# Description:

Changing URL format : submitQuery to submit-query

# How to test:

Open the app
Check that the default page is localhost:3000/submit-query
Check that this page works

# Other notes:
Tried this to have the wanted url format, would this be the solution or is it better to directly change the folders names ?
